quixstreams/dataframe/dataframe.py

```diff
@@ -1611,9 +1611,11 @@ def concat(self, other: "StreamingDataFrame") -> "StreamingDataFrame":
         """
 
         merged_stream = self.stream.merge(other.stream)
-        return self.__dataframe_clone__(
+        sdf = self.__dataframe_clone__(
             *self.topics, *other.topics, stream=merged_stream
         )
+        self._registry.require_time_alignment()
+        return sdf
```

Comment:
Is there a gotcha in tweaking the registry before sdf clone?
```suggestion
        self._registry.require_time_alignment()
        return self.__dataframe_clone__(
            *self.topics, *other.topics, stream=merged_stream
        )
```